### PR TITLE
move namedpipe connector to separate 

### PR DESCRIPTION
**Pull request by colombod**

### CODE REVIEW

**Review by shyamnamboodiripad — APPROVED** [review]

File: eng/publish/PublishVSCodeExtension.ps1 [hunk 1]
```diff
@@ -47,6 +47,7 @@ try {
                 "Microsoft.DotNet.Interactive.Journey",
                 "Microsoft.DotNet.Interactive.Kql",
                 "Microsoft.DotNet.Interactive.Mermaid",
+                "Microsoft.DotNet.Interactive.NamedPipe",
```
Comment (jonsequitur) [inline eng/publish/PublishVSCodeExtension.ps1:50]:
```suggestion
                "Microsoft.DotNet.Interactive.NamedPipeConnector",
```

File: src/Microsoft.DotNet.Interactive.NamedPipeConnector/Microsoft.DotNet.Interactive.NamedPipeConnector.csproj [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```
Comment (shyamnamboodiripad) [inline src/Microsoft.DotNet.Interactive.NamedPipeConnector/Microsoft.DotNet.Interactive.NamedPipeConnector.csproj:4]: This probably should be net7.0-windows 

```suggestion
    <TargetFramework>net7.0-windows</TargetFramework>
```

File: src/Microsoft.DotNet.Interactive.NamedPipeConnector.Tests/NamedPipeConnectionTests.cs [hunk 3]
```diff
@@ -5,15 +5,18 @@
...
 using Microsoft.DotNet.Interactive.Connection;
 using Microsoft.DotNet.Interactive.CSharp;
 using Microsoft.DotNet.Interactive.FSharp;
+using Microsoft.DotNet.Interactive.Tests;
 
+using Xunit;
 using Xunit.Abstractions;
 
-namespace Microsoft.DotNet.Interactive.Tests;
+namespace Microsoft.DotNet.Interactive.NamedPipeConnectorConnector.Tests;
```
Comment (jonsequitur) [inline src/Microsoft.DotNet.Interactive.NamedPipeConnector.Tests/NamedPipeConnectionTests.cs:19]:
```suggestion
namespace Microsoft.DotNet.Interactive.NamedPipeConnector.Tests;
```

File: src/Microsoft.DotNet.Interactive.NamedPipeConnector/ConnectNamedPipeCommand.cs [hunk 4]
```diff
@@ -5,8 +5,9 @@
 using System.CommandLine;
 using System.CommandLine.Invocation;
 using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Connection;
 
-namespace Microsoft.DotNet.Interactive.Connection;
+namespace Microsoft.DotNet.Interactive.NamedPipeConnectorConnector;
```
Comment (jonsequitur) [inline src/Microsoft.DotNet.Interactive.NamedPipeConnector/ConnectNamedPipeCommand.cs:10]:
```suggestion
namespace Microsoft.DotNet.Interactive.NamedPipeConnector;
```

**Review by jonsequitur — APPROVED** [review]

**Pull request merged**